examples/decorator/src/lib.rs

```diff
@@ -1,6 +1,6 @@
 use pyo3::prelude::*;
 use pyo3::types::{PyDict, PyTuple};
-use std::cell::Cell;
```

Comment:
There's some narrative text around this example in the guide explaining why it uses `Cell`. I should probably update that text to instead discuss why it uses an atomic.

Reply:
Since it's not actually necessary to change this I'm going to hold off on editing the docs until we have a consensus that we want to fix thread safety issues in the examples.

Reply:
Why don't we do that, feel free to steal this code though! I'll go ahead and back this out along with the rest of the changes to the examples.